modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -54,13 +54,17 @@ ifdef::google-cloud-platform[]
 +
 [source,terminal]
 ----
-$ ccoctl gcp delete --name=<name> --project=<gcp_project_id>
+$ ccoctl gcp delete \
+--name=<name> \
+--project=<gcp_project_id> \
+--credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>
 ----
 +
 where:
 +
 ** `<name>` matches the name used to originally create and tag the cloud resources.
 ** `<gcp_project_id>` is the GCP project ID in which cloud resources will be deleted.
+** `<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the files for the component `CredentialsRequest` objects.
```

Comment:
There's a good chance that the `CredentialsRequests` from installation won't be available so I think we should link to how to extract them for $version from the installation steps.